test/test.js

```diff
@@ -137,6 +137,17 @@ describe('ScssCommentParser', function () {
       });
     });
 
+    describe('css rule', function(){
+      it('should detect it', function(){
+        var context = parser.contextParser(getContent('rule.test.scss'));
+        assert.deepEqual(context, {
+          type: 'css',
+          name: '.foo',
+          value: 'font-weight: bold;\n  color: red;\n  .bar {\n    color: blue;\n  }'
```

Comment:
Yeah, I debated about the names. I chose `name` and `value` for consistency with all the other types, but if that consistency doesn't matter it would probably be better to use `selector` and `body`.

And the leading `{` and trailing `}` of the body are both removed, but if you think it's better to include them I can certainly do that.